optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/ProblemBenchmarkResult.java

```diff
@@ -220,6 +222,20 @@ public boolean hasProblemStatisticType(ProblemStatisticType problemStatisticType
         return false;
     }
 
+    public boolean hasSubSingleCountBiggerThanOne() {
+        if (hasNonDefaultSubSingleCount != null) {
+            return hasNonDefaultSubSingleCount;
+        }
+        for (SingleBenchmarkResult singleBenchmarkResult : singleBenchmarkResultList) {
+            if (singleBenchmarkResult.getSolverBenchmarkResult().getSubSingleCount() > 1) {
+                hasNonDefaultSubSingleCount = true;
```

Comment:
This calculation should happen indirectly through PlannerBenchmarkResult#accumulateResults(...), so somewhere during accumulateResults(). It should not happen in the lazy getter.